### DISCUSSION TITLE
Add live chat support

### DISCUSSION THREAD

**Discussion by corysimmons** [hcengineering/platform#4594] [Ideas]
Something like Intercom that turns live chats into tickets.

**Comment by aplatoff** [comment]
Hi Cory,

Thank you for your valuable suggestion regarding the addition of live chat support, akin to Intercom, with the capability to convert live chats into tickets. This aligns well with our vision for enhancing user support and integrating team collaboration tools, such as Slack, to streamline the support process.

We recognize the challenges teams face when using disjointed products for customer support and engineering tasks, which can sometimes complicate processes rather than streamline them. Our aim is to create a fully integrated workflow that connects customers, engineering, and other teams seamlessly. Products like [Pylon](https://usepylon.com) are setting a great example in this space.

We are keen to explore this feature further and would appreciate any specific insights you might have. For instance, your thoughts on the necessity of Slack integration or the benefits of an Intercom-like application would be invaluable. Additionally, integrating GitHub discussions as part of our broader collaboration with GitHub is something we are considering as a beneficial enhancement.

We look forward to hearing more from you on this topic and will keep you updated on our progress. In the meantime, please feel free to share any further specifics or requirements you envision for this feature.

Thank you once again for your input! -- Andrey

**Reply by corysimmons** [reply]
So Intercom and Zendesk, etc. all have some interface where you have to login to their website then you'll get browser notifications when someone sends a message. Then you (a customer support tech) will reach some conclusion, like "Customer discovered a bug in XYZ".

99% of the time it's something legit and you need to add it to a backlog.

There are some crappy integrations for ZenDesk -> Jira Ticket, but it's 3rd party and it would just be ideal if there was a swimlane like "From Customer Support" or something that Project Managers could clean up and put into the Backlog and eventually a Sprint.

If you're platform is meant to be all-in-one, this has been a huge aspect of day-to-day development at a few companies I've worked at, and it'd be nice if it was just built in.

**Reply by aplatoff** [reply]
Cool, let's do it! 👍

**Comment by uhlhosting** [comment]
Maybe an integration @typebot could solve such issues.
